app/build.gradle

```diff
@@ -77,7 +77,6 @@ android {
             signingConfig signingConfigs.debug
             applicationIdSuffix '.debug'
             versionNameSuffix "-debug"
-            shrinkResources true
```

Comment:
Thanks for useful information.
I think it's not good but is small issue for me 😉